nikola/nikola.py

```diff
@@ -1306,6 +1306,7 @@ def render_template(self, template_name, output_name, context, url_type=None):
         for k in self._GLOBAL_CONTEXT_TRANSLATABLE:
             local_context[k] = local_context[k](local_context['lang'])
         local_context['is_rtl'] = local_context['lang'] in LEGAL_VALUES['RTL_LANGUAGES']
+        local_context['url_type'] = self.config['URL_TYPE'] if url_type is None else url_type
```

Comment:
I couldn't find a reason why anyone calling this function does NOT want to have `url_type` in context (except that it causes all pages to recompile, that is).